### ISSUE TITLE
Kodi crashes after start when updating library on start is activated

### ISSUE THREAD

**Issue by jfwerner** [xbmc/metadata.themoviedb.org.python#144]
When I open Kodi 19.4 on Windows 10, it starts to scan the folders, but after a few seconds. crashes.

**Comment by jfwerner** [comment]
@KarellenX 

No, only happens when using this add-on. Non-python version works fine. I can provide dumps/logs if you want

**Comment by KarellenX** [comment]
Yes, provide the debug log and I will have a look. Make sure the log captures the crashing. Though a lot of the times the crashing occurs too quickly for the actual cause to be logged, but we might get lucky.

**Comment by KarellenX** [comment]
>As you probably guessed the "on start" part of my report is irrelevant. It happens on a manual library update as well.

It is relevant as we can glean a whole lot from it. Hardware, os versions, audio and graphic drivers, addons installed.

Please include the full log and post it to https://paste.kodi.tv/ and provide the link.

**Comment by jfwerner** [comment]
Yes. Does my file not include that? I restarted Kodi after turning the logger on, it also says "2022-04-27 19:50:59.568 T:13700    INFO <general>: Starting Kodi (19.4 (19.4.0) Git:20220302-e12e66e019). Platform: Windows NT x86 64-bit" on the 2nd line so it doesn't look like it missed it.

**Comment by jfwerner** [comment]
I can post it again in a different form if you want...
https://paste.kodi.tv/oyoposopeq.kodi

**Comment by KarellenX** [comment]
Got it. I don't know about your other logs, I didn't look at them. I avoid downloading files as it is too much hassle- download, unzip virus check, open, then delete. Try doing that 30+ times per day. Too much work, when I can click on a simple link and stay safe.

Nothing revealing in the logs. But this is where support stops. I am not going look into it further here.
Please ask your question on the forum and we can continue there.

Thanks.

**Comment by jfwerner** [comment]
It is not a question lol, good luck fixing your code. If you want people to file bugs properly, post up some guidelines.

**Comment by KarellenX** [comment]
We judge the reports individually, that is our guideline. Most times the real bugs jump out and are easily discerned. Others we rely on trends. Your report fits neither.

I know this is not a bug because there are millions of users using this scraper, including me and most of the other Kodi team members, and you are the only one reporting this error. It is something in your setup, which I was quite happy to try and delve into deeper _on the forum._

At this stage there is nothing to "fix".

But your log is here and if the addon developer spots something he will chime in.

Regards.
